components/message/index.en-US.md

```diff
@@ -14,6 +14,17 @@ Display global messages as feedback in response to user operations.
 
 ## API
 
+- `message.open(config)`
```

Comment:
I think we don't need to expose a new `message.open`, just adding `icon` to the original `Model.methold` would be enough.